src/DotNet/DotNet.csproj

```diff
@@ -75,8 +80,9 @@
       <_WorkloadSource Include="$(PackageOutputPath)" />
       <_WorkloadSource Include="@(_AllNuGetSources)" />
     </ItemGroup>
+    <Exec Command="dotnet nuget add source &quot;$(PackageOutputPath)&quot; --name artifacts --configfile &quot;$(MauiRootDirectory)NuGet.config&quot;" />
```

Comment:
If you're going to commit this change to dotnet/maui/main, this modifies the file and leaves it. So running `git status` would show changes.

Can we copy the `NuGet.config` to a temporary location like `$(DotNetTempDirectory)` and use that one? Then delete it afterward?

Reply:
fixed!